src/main/scala/au/com/agiledigital/toolform/tasks/generate/docker/GenerateDockerComposeV3.scala

```diff
@@ -0,0 +1,175 @@
+package au.com.agiledigital.toolform.tasks.generate.docker
+
+import java.io.{BufferedWriter, FileWriter}
+
+import au.com.agiledigital.toolform.app.{ToolFormConfiguration, ToolFormError}
+import au.com.agiledigital.toolform.model._
+import au.com.agiledigital.toolform.tasks.generate.{WriterContext, YamlWriter}
+import au.com.agiledigital.toolform.tasks.generate.docker.DockerFormatting._
+import au.com.agiledigital.toolform.tasks.generate.docker.SubEdgeDef.subEdgeDefsFromProject
+import au.com.agiledigital.toolform.version.BuildInfo
+
+import scalaz.Scalaz._
+import scalaz._
+
+class GenerateDockerComposeV3() extends YamlWriter {
```

Comment:
Add comments to this and `runDockerComposeV3`. Top-level types and the method that serves as the main entry point into an instance should be well documented.

Reply:
Done